apps/server/src/modules/group/repo/group.repo.ts

```diff
@@ -205,4 +163,39 @@ export class GroupRepo extends BaseDomainObjectRepo<Group, GroupEntity> {
 
 		return domainObjects;
 	}
+
+	private buildPipeline(skip?: number, limit?: number, nameQuery?: string): unknown[] {
+		let nameQueryMatch = {};
```

Comment:
I would prefer the name "nameRegexFilter" becasue it only stores the filter for the name query